man/dockerd.8.md

```diff
@@ -151,7 +151,7 @@ $ sudo dockerd --add-runtime runc=runc --add-runtime custom=/usr/local/bin/my-ru
   Set parent cgroup for all containers. Default is "/docker" for fs cgroup
   driver and "system.slice" for systemd cgroup driver.
 
-**--config-file**="/etc/docker/daemon.json"
+**--config-file**=*"/etc/docker/daemon.json"*
```

Comment:
1. This is wrong to put a default value (like done here) or an example value in synopsis. The proper way would be to say something like

```markdown
**--config-file**=*"path"*
```

and add
```markdown
Default is  */etc/docker/daemon.json*.
```
to the option description.

Reply:
To explain the last part -- while a file name (such as `/etc/docker/daemon.json`) is a literal, the man page convention is to typeset it as italic.